conf/medair.config

```diff
@@ -0,0 +1,39 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'Cluster profile for medair (local cluster of Clinical Genomics Gothenburg)'
+  config_profile_contact = 'Clinical Genomics, Gothenburg'
+  config_profile_url = 'https://www.scilifelab.se/units/clinical-genomics-goteborg/'
+}
+
+//Nextflow parameters
+singularity {
+    enabled = true
+}
+
+profiles {
+  
+  wgs {
+    process {
+      queue = 'wgs.q'
+      executor = 'sge'
+      penv = 'mpi'
+      process.clusterOptions = '-l excl=1'
+      params.max_cpus = 40
+      params.max_time = 48.h
+      params.max_memory = 128.GB
+    }
+  }
+
+  production {
+    process {
+      queue = 'production.q'
+      executor = 'sge'
+      penv = 'mpi'
+      process.clusterOptions = '-l excl=1'
+      params.max_cpus = 40
+      params.max_time = 480.h
+      params.max_memory = 128.GB
+    }
+  }
+}
+
```

Comment:
Är det inte värt att ha path till sentieon container med i denna config filen?  
process {
    withLabel:'sentieon' {
        container = "/apps/bio/singularities/sentieon-211204-peta.simg"
    }
}

Reply:
That's a good idea. I didn't find any example of another cluster config that does it. But in the sarek specific config for the munin cluster it was done: `conf/pipeline/sarek/munin.config`
What do we think? Should that go to the general config or to a nf-core/raredisease specific one?

Reply:
I'd actually like to have it in the cluster config file which I assume we will use for the most of Nextflow pipelines and also test runs. So I think it's better to already have it here than specifying it in other configs where we probably want to specify some input/output paths and other variables. (But I know that not in all of the pipelines we run sentieon )

Reply:
Hum - what I meant is that one can have a cluster & pipeline specific config, that is also stored in nf-core/configs. But I agree, it does not harm to have it there even though not all pipelines use Sentieon.
What about the Sentieon version? Should we go with the more recent version as you suggest, or start with the version used in WOPR and change once we have done the comparisons?

Reply:
Another question, do we want to call it medair?

Reply:
I think medair is a suitable name. It's overall what we call the cluster